0000-template.md

```diff
@@ -1,51 +1,48 @@
+- Title: (fill me in with a human-focussed title for the RFC)
 - Feature Name: (fill me in with a unique ident, `my_awesome_feature`)
 - Start Date: (fill me in with today's date, YYYY-MM-DD)
-- RFC PR: [rust-lang/rfcs#0000](https://github.com/rust-lang/rfcs/pull/0000)
+- RFC PR: [#0000](https://github.com/rust-lang/rfcs/pull/0000)
 - Rust Issue: [rust-lang/rust#0000](https://github.com/rust-lang/rust/issues/0000)
+- Team: (fill me in with the team or teams responsible for this RFC)
+- Keywords: (fill me in with keywords which may be useful when searching for this RFC)
+- Previous RFCs: #0000 (fill me in with RFC numbers for any RFCs this RFC supersedes, deprecates, or extends)
```

Comment:
I suggest if these are intended to be links to make them a link in the template. Having unclickable numbers could be frustrating.

```suggestion
- Previous RFCs: [#0000](https://github.com/rust-lang/rfcs/pull/0000) (fill me in with RFC numbers for any RFCs this RFC supersedes, deprecates, or extends)
```

Reply:
Ah, I was hoping GH would render these as it does in issues. It seems not, so links would be better. Good catch!